### PR TITLE
feat(sprint-details): Deleted details and Updated Status btn.

### PR DESCRIPTION
**Pull request by AvinashMahanthi**
Signed-off-by: AvinashMahanthi <avinashfab5678@gmail.com>

### Functionality:
Remove the Sprint Details and update the Sprint Status button.

### Solution:

- Updated the Sprint status to the dropdown button.
- Removed the Sprint Details row.

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:

- ng serve
- firebase emulators:start
- Go to Board.

### Screenshots:
image

image

image

### CODE REVIEW

**Review by shivamgoyal7 — APPROVED** [review]
Ship it

File: src/app/body/board/sprint-details/sprint-details.component.html [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+                        <div class="dropdown-menu">
+                            <div class="dropdown-item" type="button" (click)="updateSprintStatus('Under progress')">Under progress</div>
+                            <div class="dropdown-item" type="button" (click)="updateSprintStatus('Completed')">Completed</div>
+                        </div>
+                    </div>
+                    <span class="material-icons align-middle m-2" *ngIf="Status === 'Completed' " style="vertical-align:baseline;">check_circle</span>
+                </div>
+            </div>
+        </div>
+        <div class="card-body" style="background-color:#202124;">
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.html:37]: Please move all style tag in .css

File: src/app/body/board/sprint-details/sprint-details.component.html [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+        <div class="card-header" style="font-weight: bold; background-color: #5A20F0; color:white;">
+            <div class="row  align-items-center">
+                <div class="col-md-2 col-4">
+                    <h5 style="font-size: 18px;">Sprint No:</h5>
+                </div>
+                <div class="col-md-7 col-5">
+                    <h5 id="sprintId">{{currentSprintName}}</h5>
+                </div>
+
+                <div class="row col-md-3 col-8 text-right">
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.html:12]: column and row merged in same line is not a right approach.
Please fix it

File: src/app/body/board/sprint-details/sprint-details.component.html [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
+                    <div class="mr-1">
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.html:13]: just for margin a div tag with class, we can give padding once column is implemented.

File: src/app/body/board/sprint-details/sprint-details.component.html [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
+                        <div class="dropdown">
+                            <button class="btn dropdown-toggle" type="button" id="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Filter</button>
+                            <div class="dropdown-menu dropdown-menu-right px-4 py-3" id="filterMenu" aria-labelledby="dropdownMenuButton">
+                                <div class="row py-3">
+                                    <input class="col-6" style="font-size: 11px;" type="number" id="filterSprintNumberInput" [(ngModel)]="filterSprintNumber">
+                                    <div class="col-6 pl-2">
+                                        <button class="btn" id="dropdownButton" style="font-size: 14px;" (click)="changeSprintNumber()">Filter</button>
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.html:20]: Please move style tag in .css

File: src/app/body/board/sprint-details/sprint-details.component.html [hunk 5]
```diff
@@ -1,55 +1,55 @@
...
-            </div>
-            <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Status: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;">{{Status}}</h6>
-            </div>
-        </div>
-    </div>
+<div class="pt-3 text-white">
+    <div class="card" style="border: none;">
+        <div class="card-header" style="font-weight: bold; background-color: #5A20F0; color:white;">
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.html:3]: Please move style tag into .css

File: src/app/body/board/sprint-details/sprint-details.component.html [hunk 6]
```diff
@@ -1,55 +1,55 @@
...
+            <div class="row py-2">
+                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">End Date: </h6>
+                <h6 class="card-text col-md-10 col-8" id="sprintEndDate" style="font-size: 12px;">{{EndDate}}
+                </h6>
+            </div>
+            <div class="row py-2">
+                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Status: </h6>
+                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;">{{Status}}</h6>
+            </div>
+        </div>
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.html:53]: Move all the style tags in .css using common class.

File: src/app/body/board/sprint-details/sprint-details.component.ts [hunk 7]
```diff
@@ -23,6 +23,7 @@ export class SprintDetailsComponent implements OnInit {
   componentName :string="SPRINT-DETAILS"
   filterSprintNumber: number;
   sprintStatus: string;
+  enableLoader: boolean = false;
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.ts:26]: Loader is being implemented but not available in html, call the loading spinner in css until the api call is completed.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please complete the required changes asap.

**Comment by viv9k** [comment]
resolve merge conflict and adapt your code with new theme based implementation.

**Pull request closed without merging**